src/main/resources/com/google/api/codegen/java/main.snip

```diff
@@ -380,27 +405,64 @@
 @end
 
 @private callableMethodImpl(apiMethod)
-  @if apiMethod.isStreaming
-    {@streamingCallableMethod(apiMethod)}
-  @else
-    {@nonStreamingCallableMethod(apiMethod)}
-  @end
+  public final UnaryCallable<{@apiMethod.serviceRequestTypeName}, \
+      {@apiMethod.responseTypeName}> {@apiMethod.name}() {
+    return {@apiMethod.callableMethod.callableName};
+  }
 @end
 
-@private nonStreamingCallableMethod(apiMethod)
-  public final UnaryCallable<{@apiMethod.apiRequestTypeName}, \
+@private streamingCallableMethod(apiMethod)
+  @let coreSampleCode = streamingCallableMethodSampleCode(apiMethod)
```

Comment:
I've used a `@let` for this particular scenario from the beginning. In this scenario, we need to fully render a sub-snippet and then add stars to every line of it; using a `@let` is the only way to do it.